containers/SynonymList.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { changeSynonyms } from '../actions'
+
+let SynonymList = ({ entityId, refId, synonyms, dispatch }) => (
```

Comment:
It's actually a list. The format for synonym is "s1; s2; s3". For now, it's only one input field, but it might be changed according to the layout design.